### PR TITLE
update build tools, fabric, and okhttp

### PR DESCRIPTION
**Pull request by harkin**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -168,8 +168,7 @@ dependencies {
     androidTestCompile 'com.android.support.test:runner:0.5'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
     apt 'org.projectlombok:lombok:1.16.10'
-    onlineCompile 'com.squareup.okhttp3:okhttp:3.4.1'
-    onlineCompile 'com.squareup.okio:okio:1.10.0'
```
Comment (harkin) [inline app/build.gradle]: okio is a [transitive dependency](https://github.com/square/okhttp/blob/master/okhttp/pom.xml#L17) of okhttp so specifying it here doesn't achieve much

**Comment by harkin** [comment]
Ci is failing with 
```
* What went wrong:
A problem occurred configuring project ':app'.
> You have not accepted the license agreements of the following SDK components:
  [Android SDK Build-Tools 26.0.2].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
```
Someone will need to accept that?

**Comment by MARTINI1** [comment]
in this PR travis use build tools 26.0.2 but then in PR # 790 it is changed to 26.0.1, it also happens in buildToolsVersion that in one PR is updated but then it goes back to 25.0.0

please fix these changes

**Comment by harkin** [comment]
@MARTINI1 Not sure what you mean. This PR updates the android build tools to `26.0.2`, the latest version ([release notes](https://developer.android.com/studio/releases/build-tools.html)). My other PR updates the android platform tools to `26.0.1`, the latest version ([release notes](https://developer.android.com/studio/releases/platform-tools.html))

They are independent, atomic changes. Neither PR is resetting something done in the other. What needs fixing?

**Comment by MARTINI1** [comment]
you're right, I was confused when I looked at the lines of build-tools and platform-tools...

now only remains @nahojjjen merge this PR, #790 and the #792 

sorry for the confusion

**Comment by nahojjjen** [comment]
Looks good 👍

**Pull request merged**